test/Sleep-test.js

```diff
@@ -0,0 +1,36 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Sleep = require('../src/Sleep');
+// const SleepRepo = require('../src/SleepRepo');
+
+describe('Hydration', () => {
+  let sleepData, instanceData;
+
+  beforeEach(() => {
+    sleepData = {
+        "userID": 1,
+        "date": "2019/06/15",
+        "hoursSlept": 6.6,
+        "sleepQuality": 2.5
+      };
```

Comment:
Ima be super nit-picky here for a second. Line 16 can have two less spaces in its indentation. Most of the time the ending curly brace will line up with the beginning of the object's name.